common/containers/Tabs/SendTransaction/components/AmountField.jsx

```diff
@@ -32,7 +34,7 @@ export default class AmountField extends React.Component {
           />
           <UnitDropdown
             value={unit}
-            options={['ether']}
+            options={['ether'].concat(this.props.tokens)}
```

Comment:
Yep, it is that way so we dont forget to support ETC/RSK or whatever else we need to support.